src/instructlab/eval/mmlu.py

```diff
@@ -180,19 +204,15 @@ def run(self) -> tuple:
 
         individual_scores: dict = {}
         agg_score: float = 0.0
-        model_args = f"pretrained={self.model_path},dtype={self.model_dtype}"
-
-        tm = TaskManager(verbosity="DEBUG", include_path=self.sdg_path)
 
-        mmlu_output = simple_evaluate(
-            model="hf",
-            model_args=model_args,
-            tasks=self.tasks,
-            num_fewshot=self.few_shots,
-            batch_size=self.batch_size,
-            task_manager=tm,
+        results = results = run_mmlu(
```

Comment:
@danmcp should this just be `results = run_mmlu(...)` ?

Reply:
Oops, yes!  Fixed.